packages/gafl-webapp-service/src/pages/contact/address/lookup/address-lookup.njk

```diff
@@ -10,12 +10,12 @@
 
 {% set errorMap = {
         'premises': {
-            'string.empty': { ref: '#premises', text: 'You did not enter your building number or name' },
-            'string.max': { ref: '#premises', text: 'Your building number or name must be less than 50 characters' }
+            'string.empty': { ref: '#premises', text: 'Enter the house number or name' },
```

Comment:
Inconsistent - we use "building number or name" above and "house number or name" here

Reply:
Yes - its because the UCD says house number but the historical standard for addresses in EA is building number or name. I will ask UCD which one we are going for.

Reply:
Confirmed with UCD; Building number